From 1687a5451eacb5b2661725ed5ded8582d73c798f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Fri, 19 Apr 2019 09:02:00 +0300
Subject: [PATCH 002/152] altTab: use AltTab prefix on AppIcon class

There's an AppIcon class on appDisplay, so it won't be easy to discern
the one we're going to add in altTab.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/700
---
 js/ui/altTab.js | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/js/ui/altTab.js b/js/ui/altTab.js
index 9d884cef5..55c680877 100644
--- a/js/ui/altTab.js
+++ b/js/ui/altTab.js
@@ -648,8 +648,9 @@ class WindowCyclerPopup extends CyclerPopup {
     }
 });
 
-var AppIcon = GObject.registerClass(
-class AppIcon extends St.BoxLayout {
+var AppIcon = GObject.registerClass({
+    GTypeName: 'AltTab_AppIcon'
+}, class AppIcon extends St.BoxLayout {
     _init(app) {
         super._init({ style_class: 'alt-tab-app',
                       vertical: true });
-- 
2.23.0

